docs/guide/learning/how-to-learn-vue.md

```diff
@@ -195,6 +195,29 @@ In this section, we will go over the most popular technologies and the struggles
 
 ### AngularJS (Angular 1.x)
 
+**Similarities**
+
+- Templates are pure HTML
```

Comment:
I would rephrase that they both use HTML based templating systems. The directives and what knot are technically not "pure" hehe, you get my idea.

Reply:
Done!